integrations/terraform-cloud/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,69 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: workspace
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .attributes.name
+          blueprint: '"terraformWorkspace"'
+          properties:
+            organization : .relationships.organization.data.id
+            createdAt: .attributes."created-at"
+            updatedAt: .attributes."updated-at"
+            terraformVersion: .attributes."terraform-version"
+            locked: .attributes.locked
+            executionMode: .attributes."execution-mode"
+            resourceCount: .attributes."resource-count"
+            latestChangeAt: .attributes."latest-change-at"
+          relationship:
+            terraformCloudStateVersion: .relationships.current-state-version.data.id

```

Comment:
The current-state-version is not in quote?

Reply:
It should be. 

Resolved